mobsf/StaticAnalyzer/tests.py

```diff
@@ -24,6 +25,12 @@ def static_analysis_test():
         uploaded = []
         logger.info('Running Upload Test')
         http_client = Client()
+
```

Comment:
This is not required. MobSF in CI runs without user authentication using `MOBSF_DISABLE_AUTHENTICATION` environment variable.